server/models.js

```diff
@@ -1,4 +1,6 @@
 const db = require('../database/index.js');
+const axios = require('axios');
+const config = require('../config.js');
```

Comment:
If you destructure this line like `const { placesKey } = require('../config.js');` you can reference the key as just 'placesKey' rather than 'config.placesKey' which I think is a little cleaner/more intuitive.